curriculum/challenges/english/08-coding-interview-prep/rosetta-code/averages-mode.english.md

```diff
@@ -6,7 +6,9 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<p>Write a program to find the <a href="https://en.wikipedia.org/wiki/Mode (statistics)" title="wp: Mode (statistics)">mode</a> value of a collection.</p><p>The case where the collection is empty may be ignored. Care must be taken to handle the case where the mode is non-unique.</p><p>If it is not appropriate or possible to support a general collection, use a vector (array), if possible. If it is not appropriate or possible to support an unspecified value type, use integers.</p>
+Write a program to find the <a href='https://en.wikipedia.org/wiki/Mode (statistics)' title='wp: Mode (statistics)' target='_blank'>mode</a> value of a collection.
+The case where the collection is empty may be ignored. Care must be taken to handle the case where the mode is non-unique.
+If it is not appropriate or possible to support a general collection, use a vector (array), if possible. If it is not appropriate or possible to support an unspecified value type, use integers.
 </section>
 
 ## Instructions
```

Comment:
@scissorsneedfoodtoo I think the instructions are mistakenly moved to description here.

Reply:
@Nirajn2311, thank you for taking the time to go through all of these changes!

Quite a few of the challenges have very short instructions. If the instructions are just a few sentences long like they are here, I would usually leave them in the description. While I would prefer to move all the text to the instructions section, the empty description would still render a `<pre>` element, so the text would have dividing lines above and below it. Here's a screen shot:

![image](https://user-images.githubusercontent.com/2051070/55045935-15e86380-5083-11e9-9c87-9c1d48f86ce4.png)

The exception would be if there's a sentence toward the end of the description that reads something like, "Implement a function to return the..." I would move sentences/paragraphs like that to the instructions section.